### PR TITLE
fix: a11y for markdown but mostly due to reminder

### PR DESCRIPTION
**Pull request by vincentaudebert**
## Type

- Enhancement

#### What is expected?

a11y for reminder but only impacts markdown component

#### The following changes where made:

1. removes span for inline mode to avoid having p tag inside a span tag.

### CODE REVIEW

File: src/components/MarkDown/index.tsx [hunk 1]
```diff
@@ -61,11 +61,7 @@ const RootRendererComponent: FunctionComponent<{
   inline?: boolean
   parentProps?: XStyledProps
 }> = ({ inline = false, parentProps = {}, children }) => (
-  <StyledContainer
-    as={inline ? 'span' : 'div'}
-    inline={inline}
-    {...parentProps}
-  >
+  <StyledContainer as="div" inline={inline} {...parentProps}>
```
Comment (chambo-e) [inline src/components/MarkDown/index.tsx:64]: So this will always be a div now ? 

This is going to break a lot of Markdown components :)

Comment (vincentaudebert) [reply src/components/MarkDown/index.tsx:64]: it should not

If inline prop is on, we css p tags to be inlined. It should be enough. We need to check if we have some places with multiple markdowns following each other but that's it?

In any case this is required as p tag inside a span tag isn't correct :(

**Review by chambo-e — APPROVED** [review]

File: src/components/MarkDown/index.tsx (hunk 1 above)
Comment (chambo-e) [reply src/components/MarkDown/index.tsx:64]: 👌 

Btw the as="div" is useless, Box is by default a div

Comment (vincentaudebert) [reply src/components/MarkDown/index.tsx:64]: Good point I fix it

**Review by matthprost — APPROVED** [review]

**Pull request merged**